messages/sam_msgs/msg/Topics.msg

```diff
@@ -12,7 +12,7 @@ string LCG_CMD_TOPIC = 'core/lcg_cmd'
 string THRUSTER1_CMD_TOPIC = 'core/thruster1_cmd'
 string THRUSTER2_CMD_TOPIC = 'core/thruster2_cmd'
 string THRUST_VECTOR_CMD_TOPIC = 'core/thrust_vector_cmd'
-
+string RPM_CMD_TOPIC = 'core/rpm_cmd'
```

Comment:
This is new to me: Does sam not use thrusterX_cmd topics anymore and uses just this now?